### PR TITLE
Specify NPM version

### PR DESCRIPTION
**Pull request by shawnrushefsky**
This is fixing a build error, in combination with https://github.com/igroff/sockjs-node/pull/1 (which has been merged). Running on my devship: https://github.com/glg/ec2.starphleet.dev.headquarters/blob/ShawnRushefsky/epi-ht-canary/orders#L94

```
[info] 10/07/2020 02:30:27 PM -     order             : epi-ht-canary
[info] 10/07/2020 02:30:27 PM -     latest status     : online
[info] 10/07/2020 02:30:27 PM -     latest active     : epi-ht-canary-721a5e3-5f2c2d8
[info] 10/07/2020 02:30:27 PM -     git author        : shawn.rushefsky@gmail.com
[info] 10/07/2020 02:30:27 PM -     git sha           : 5f2c2d8
[info] 10/07/2020 02:30:27 PM -     latest git msg    : Merge branch 'master' into specify-github
[info] 10/07/2020 02:30:27 PM -     
[info] 10/07/2020 02:30:27 PM -     Container Details : 
[info] 10/07/2020 02:30:27 PM -     ------------
[info] 10/07/2020 02:30:27 PM -     epi-ht-canary-721a5e3-5f2c2d8 - [10.0.92.244:8080] - [online]
```

part of glg/metadevops-issues#470

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "q": "0.9.2",
     "simplog": "0.2.2",
     "snowflake-sdk": "1.5.3",
-    "sockjs": "igroff/sockjs-node#pass-headers",
+    "sockjs": "github:igroff/sockjs-node#pass-headers",
```
Comment (fearphage) [inline package.json:37]: Does `npm` add this by default now or is that you?

File: package.json [hunk 2]
```diff
@@ -45,7 +45,8 @@
     "watch": "make watch"
   },
   "engines": {
-    "node": "12.3.1"
+    "node": "12.3.1",
+    "npm": "6.9.2"
```
Comment (fearphage) [inline package.json:49]: Is this the latest? We're never upgrading this again (probably) until something breaks so can we grab the latest?

**Review by fearphage — CHANGES_REQUESTED** [review]
A question and a suggestion

File: package.json (hunk 1 above)
Comment (shawnrushefsky) [reply package.json:37]: I did that. I think it's optional, but I find it more explicit and clear

File: package.json (hunk 2 above)
Comment (shawnrushefsky) [reply package.json:49]: Yeah, I'll make sure it still builds with a later npm

**Review by igroff — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (shawnrushefsky) [reply package.json]: ok, confirmed builds and passes healthcheck with npm 6.14.8

**Review by fearphage — APPROVED** [review]

**Pull request merged**